docs/source/algorithms/CreateTransmissionWorkspaceAuto-v2.rst

```diff
@@ -0,0 +1,114 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm is a facade over :ref:`algm-CreateTransmissionWorkspace`. It pulls numeric parameters
+out of the instrument parameter file where possible. You can override any of these automatically applied
+defaults by providing your own values.
+
+The first transmission run is the only mandatory property. If :literal:`ProcessingInstructions` is not set its value is inferred from other properties:
+
+* If :literal:`AnalysisMode = PointDetectorAnalaysis` and :literal:`PointDetectorStart = PointDetectorStop` then :literal:`ProcessingInstructions` is set to :literal:`PointDetectorStart`.
+* If :literal:`AnalysisMode = PointDetectorAnalaysis` and :literal:`PointDetectorStart != PointDetectorStop` then :literal:`ProcessingInstructions` is set to :literal`PointDetectorStart:PointDetectorStop`.
```

Comment:
Typo - :literal:`PointDetectorStart:PointDetectorStop` - it displays incorrectly